source/nodejs/adaptivecards-react/src/adaptive-card.tsx

```diff
@@ -124,7 +124,7 @@ export const AdaptiveCard = ({
         try {
             card.parse(payload);
             const result = card.render() as HTMLElement;
-            const trustedHtml = window.trustedTypes?.emptyHTML ?? "";
+            const trustedHtml = (typeof window === 'undefined') ? "" : (window.trustedTypes?.emptyHTML ?? "");
```

Comment:
any reason we can't do this (as you had originally fixed it)?

```suggestion
            const trustedHtml = window?.trustedTypes?.emptyHTML ?? "");
```

Reply:
`window?` doesn't work, it throws exception. have to use typeof

![image](https://user-images.githubusercontent.com/6290692/191333381-20595387-ee3f-4ddb-a65a-7a5a3d163e32.png)

Reply:
but `?` works fine in an existing object. For example, 
![image](https://user-images.githubusercontent.com/6290692/191333918-a53b9b4f-cb34-4c55-86c2-37f77a42dc94.png)

window is on the toplevel, and `?` doesn't work